blob/src/main/java/com/bazaarvoice/emodb/blob/db/s3/S3RateLimiter.java

```diff
@@ -0,0 +1,160 @@
+package com.bazaarvoice.emodb.blob.db.s3;
```

Comment:
Is this from the incremental stash generator? Just curious :)

Reply:
yep https://github.com/bazaarvoice/emodb-incremental-stash/blob/4accbe68c11205b42a0e3d49e1912736a4bd6a35/common/src/main/java/com/bazaarvoice/emodb/incrementalstash/io/S3RateLimiter.java, but made it configurable